documentation/en/chapter_14.md

```diff
@@ -0,0 +1,971 @@
+   <td>
+        <ul>
+            <li>2001::200:3:0:0/128 dev #7</li>
+            <li>2001::200:1:0:0/128 via fe80::200:1:0:0 dev #7</li>
+        </ul> 
+    </td>
+   <td> wireless SetPosition radio 70 70 0 </td>
+    <td>70</td>
+    <td>70</td>
+    <td>0</td>
+ </tr>
+  <tr align="left">
+    <td>Node-D</td>
+   <td>
+        <ul>
+            <li>2001::200:4:0:0/128 dev #7</li>
+            <li>2001::200:1:0:0/128 via fe80::200:2:0:0 dev #7</li>
+        </ul> 
+    </td>
+   <td> wireless SetPosition radio 110 0 0 </td>
+    <td>110</td>
+    <td>0</td>
+    <td>0</td>
+ </tr>
+</table>
+</div>
+<br>
+
+ 
+We can try to send a message from node B or C to node A and it can be success because both intermediate nodes have stored a reverse route back to the originator of the RREQ (node A).
+The only thing we cannot forget is the route inside node C to reach node A is an unconfirmed as bidirectional path.
+
+
+Now we can do a new route request from different node to test the RREQ and RREP messages and printout  the routes tables.
+Lets try with route request from node B to node C and check the route tables.
+
+After carry out the request the following picture show the content of route tables, and that information means node A is the bridge between node B and node C.This basic stage can help us to understand how many RREQ are flooding the network and how we can improve the system in order to process more information that can work as a second alternative when broken routes.
+
+<img src="../ES/imple_pic/simulation_4nodesTopology.svg" alt="drawing" height="400" width="800" align=""/>
+
+
+
+##### Table 4
+<div>
+<table id="tblOne" style="width:100%;" >
+ <tr align="center">
+    <th>Nodes</th>
+    <th>Routes</th>
+ </tr>
+  <tr align="left">
+    <td>Node-A</td>
+    <td>
+        <ul>
+            <li>2001::200:1:0:0/128 dev #7</li>
+            <li>2001::200:2:0:0/128 via fe80::200:2:0:0 dev #7</li>
+            <li>2001::200:3:0:0/128 via fe80::200:3:0:0 dev #7</li>
+        </ul> 
+    </td>
+ </tr>
+ <tr align="left">
+    <td>Node-B</td>
+    <td>
+        <ul>
+            <li>2001::200:2:0:0/128 dev #7</li>
+            <li>2001::200:3:0:0/128 via fe80::200:1:0:0 dev #7</li>
+        </ul> 
+    </td>
+ </tr>
+ <tr align="left">
+    <td>Node-C</td>
+   <td>
+        <ul>
+            <li>2001::200:3:0:0/128 dev #7</li>
+            <li>2001::200:2:0:0/128 via fe80::200:1:0:0 dev #7</li>
+        </ul> 
+    </td>
+ </tr>
+  <tr align="left">
+    <td>Node-D</td>
+   <td>
+        <ul>
+            <li>2001::200:4:0:0/128 dev #7</li>
+            <li>2001::200:2:0:0/128 via fe80::200:2:0:0 dev #7</li>
+        </ul> 
+    </td>
+ </tr>
+</table>
+</div>
+
+# X.6 third stage environment
+henceforth we are going to avoid __find_route__, because the AODV firmaware was created to be reactive and this feature is triggered when any node try to send __UDP__ packets to any destination and not path are found inside the rote table.
```

Comment:
```suggestion
Henceforth we are going to avoid __find_route__, because the AODVv2 firmware was created to be reactive and this feature is triggered when any node try to send __UDP__ packets to any destination and not path are found inside the rote table.
```